pkgs/development/python-modules/asap3/default.nix

```
@@ -0,0 +1,34 @@
+{
+  pkgs,
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "asap3";
+  version = "3.13.9";
+
+  build-system = [ setuptools ];
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-oi5rr5k1YhtmpHm0sbIlkycqslI/yT99qmQysUfqmmQ=";
+  };
+
+  nativeBuildInputs = [ pkgs.which ];
+
+  dependencies = [
+    numpy
+    ase
+  ];
+
+  meta = {
+    description = "ASAP - As Soon As Possible, Accelerated Simuations and Potentials";
+    homepage = "https://asap3.readthedocs.io/en/latest/index.html";
+    license = lib.licenses.gpl3Only;
```

```suggestion
    license = lib.licenses.lgpl3Only;
```
https://gitlab.com/asap/asap/-/blob/5ed2b800b24834ae959355284063d578785f0e17/pyproject.toml#L10